docs/custom_container.md

```diff
@@ -0,0 +1,86 @@
+# Customizing the Azure DevOps job container
+
+The Model training and deployment pipeline uses a Docker container
+on the Azure Pipelines agents to provide a reproducible environment
+to run test and deployment code.
+ The image of the container
+`mcr.microsoft.com/mlops/python:latest` is built with this
+[Dockerfile](../environment_setup/Dockerfile).
+
+In your project you will want to build your own
+Docker image that only contains the dependencies and tools required for your
+use case. This image will be more likely smaller and therefore faster, and it
+will be totally maintained by your team. 
+
+## Provision an Azure Container Registry 
+
+An Azure Container Registry is deployed along your Azure ML Workspace to manage models.
+You can use that registry instance to store your MLOps container image as well, or
+provision a separate instance.
+
+## Create a Registry Service Connection
+
+Create a service connection to your Azure Container Registry:
+- As *Connection type*, select *Docker Registry*
+- As *Registry type*, select *Azure Container Registry*
+- As *Azure container registry*, select your Container registry instance
+- As *Service connection name*, enter `acrconnection`
+
+## Update the environment definition
+
+Modify the [Dockerfile](../environment_setup/Dockerfile) and/or the
+[ci_dependencies.yml](../diabetes_regression/ci_dependencies.yml) Conda
```

Comment:
Would be good to explain what the conda file does. You mention what the docker container does above.

Reply:
Added:

Conda provides a [reusable environment for training and deployment with Azure Machine Learning](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-environments).
The Conda environment used for CI should use the same package versions as the Conda environment
used for the Azure ML training and scoring environments (defined in [conda_dependencies.yml](../diabetes_regression/conda_dependencies.yml)).
This enables you to run unit and integration tests using the exact same dependencies as used in the ML pipeline.